src/core/symbol/SymbolIdGenerator.ts

```diff
@@ -81,6 +82,36 @@ export class SymbolIdGenerator {
         return path;
     };
 
+    /**
+     * The namespace id based on the full alias name.
+     * @param fullName - the full namespace
+     */
+    public static namespaceId = (fullName: string): bigint => {
```

Comment:
do we need this method at all? 

There is already a method to return the path from fullname `public static generateNamespacePath = (fullName: string): bigint[]`

Reply:
The problem with generateNamespacePath is that the user doesn't know at a first look which item from the bigint[] should be used. It's not clear that the last one should be used ignoring the other ones. Also, the user requires 2 liners to resolve the alias id from the full name.

You can compare the APIs calls:

```
    const aliceAliasPaths = SymbolIdGenerator.generateNamespacePath('i.am.alice');
    const aliceAlias = aliceAliasPaths[aliceAliasPaths.length - 1]; // not super clear, user may think it should use another value.
```
VS:

```
const aliceAlias = SymbolIdGenerator.namespaceId('i.am.alice');
```

Having said that, I'm not sure `SymbolIdGenerator`, in general, should be part of the public API of the SDK. Maybe something like this is better?

```
const aliceAlias = factory.namespaceId('i.am.alice');
```
My other concern is that namespace ids, mosaic ids, amounts are all bigint. Bigint is great (a lot better than number[]) just that user may confuse at some point and pass a value for another value as the types are the same? Maybe having a MosaicId, NamespaceId bigint wrappers with methods like toBuilder() would make sense?

Reply:
`const aliceAlias = factory.namespaceId('i.am.alice')`

Why not implement it in the transaction factory?  Also this method should have a better name

Reply:
Yes, it can be implemented in the factory. I have add it, I can remove the one from SymbolIdGenerator. Do you have a better name?

Reply:
Removed SymbolIdGenerator.namespaceId, added factory.fullNameToNamespaceId